pcsx2/Pcsx2Config.cpp

```diff
@@ -35,10 +35,35 @@ void TraceLogFilters::LoadSave( IniInterface& ini )
 	IniEntry( IOP.bitset );
 }
 
+const wxChar* const tbl_SpeedhackNames[] =
+	{
+		L"mvuFlag",
+		L"InstantVU1"};
+
+const __fi wxChar* EnumToString(SpeedhackId id)
+{
+	return tbl_SpeedhackNames[id];
+}
+
+void Pcsx2Config::SpeedhackOptions::Set(SpeedhackId id, bool enabled)
+{
+	EnumAssert(id);
+	switch (id)
+	{
+		case Speedhack_mvuFlag:
+			vuFlagHack = enabled;
+			break;
+		case Speedhack_InstantVU1:
+			vu1Instant = enabled;
+			break;
+			jNO_DEFAULT;
+	}
+}
+
 Pcsx2Config::SpeedhackOptions::SpeedhackOptions()
 {
 	DisableAll();
-	
```

Comment:
extra newline

Reply:
Clang output - it got rid of the trailing whitespace (line used to be just a tab).
https://github.com/PCSX2/pcsx2/blob/678829a5b2b8ca7a3e42d8edc9ab201bf00b0fe9/pcsx2/Pcsx2Config.cpp#L41

Reply:
output from clang